test/gradient_test_F.jl

```diff
@@ -0,0 +1,120 @@
+# 2D gradient test for system 0.5*||Fq-d||_2^2
+# Single simultaneous source w/ random weights for both Array and judiVector
+# Author: Ziyi Yin, ziyi.yin@gatech.edu
+# Date: August 2020
+
+using JUDI.TimeModeling, Test, LinearAlgebra, PyPlot, Printf
+
+include("test_utils.jl")
+
+parsed_args = parse_commandline()
+
+println("Gradient test for system 0.5*||Fq-d||_2^2")
+
+println(parsed_args["nlayer"], " layers and tti: ",
+		parsed_args["tti"], " and freesurface: ", parsed_args["fs"],
+		" and isic: ", parsed_args["isic"] )
```

Comment:
`isic` has no effect on F at all, it is the Inverse scattering imaging condition and only impact the jacobian J and its adjoint, this is unnecessary cluter.